pkg/util/mount/mount.go

```diff
@@ -143,6 +142,23 @@ func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
 	return refs, nil
 }
 
+// GetMountRefsByDev finds all references to the device provided
+// by devPath; returns a list of paths.
+func GetMountRefsByDev(mounter Interface, devPath string) ([]string, error) {
+	mps, err := mounter.List()
+	if err != nil {
+		return nil, err
+	}
+	// Find all references to the device.
+	var refs []string
+	for i := range mps {
+		if mps[i].Path == devPath || mps[i].Device == devPath {
```

Comment:
We only need to check against mp.Device right?  Because with local storage, there's two layers of mounts:

1. /dev/sda1 -> /some/global/mount
2. /some/global/mount -> /podX/volumes/local/volumeName

And we only care about having multiples of the 2nd layer of mounts.  We don't want to count the 1st mount.